tests/SprykerEcoTest/Zed/Payone/Business/Payment/MethodMapper/OnlineBankTransferTest.php

```diff
@@ -147,7 +166,7 @@ public function testMapPaymentToDebit()
 
         foreach (static::DEBIT_COMMON_REQUIRED_PARAMS as $key => $value) {
             $this->assertArrayHasKey($key, $requestData);
-            $this->assertSame($value, $requestData[$key]);
+            $this->assertEquals($value, $requestData[$key]);
```

Comment:
why was it changed from assertSame to assertEquals
we should use strict comparison

Reply:
same issue for similar tests